var/spack/repos/builtin/packages/raja-perf/package.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import socket
+import re
+
+from spack.package import *
+from .camp import hip_for_radiuss_projects
+from .camp import cuda_for_radiuss_projects
+from .camp import blt_link_helpers
+
+
+class RajaPerf(CachedCMakePackage, CudaPackage, ROCmPackage):
+    """RAJA Performance Suite."""
+
+    homepage = "http://software.llnl.gov/RAJAPerf/"
```

Comment:
I cannot confirm this homepage.  Also, we prefer use of `https` protocol.

Reply:
Agreed.

Reply:
And fixed